apps/web/app/(auth)/account/keys/page.tsx

```diff
@@ -0,0 +1,64 @@
+import { MaxWidthContainer } from "@/features/page/page";
+import { getRequiredUser } from "@/lib/auth-session";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@workspace/ui/components/card";
+import { Typography } from "@workspace/ui/components/typography";
+import { createApiKey, deleteApiKey } from "./actions";
+import { ApiKeyList } from "./api-key-list";
+import { CreateApiKeyForm } from "./create-api-key-form";
+
+export default async function ApiKeysPage() {
+  await getRequiredUser();
+
+  return (
+    <MaxWidthContainer className="my-8 flex flex-col gap-6 lg:gap-10">
+      <div>
+        <Typography variant="h1">API Keys</Typography>
+        <Typography variant="p" className="text-muted-foreground mt-2">
+          Manage your API keys to access the SaveIt.now API programmatically.
+        </Typography>
+      </div>
+
+      <CreateApiKeyForm onSubmit={createApiKey} />
+      
+      <ApiKeyList onDelete={deleteApiKey} />
+
+      <Card>
+        <CardHeader>
+          <CardTitle>API Usage</CardTitle>
+          <CardDescription>
+            Use your API keys to access the SaveIt.now API endpoints:
+          </CardDescription>
+        </CardHeader>
+        <CardContent className="space-y-4">
```

Comment:
no need this, create a content/docs files that explain how to use the API keys and a docs page for each endpoint please